examples/asio/CMakeLists.txt

```diff
@@ -14,10 +14,10 @@ hunter_add_package(asio)
 find_package(asio CONFIG REQUIRED)
 
 add_executable(foo-headeronly foo.cpp)
-target_link_libraries(foo-headeronly PUBLIC asio::asio_headeronly)
+target_link_libraries(foo-headeronly PUBLIC asio::asio_headeronly Threads::Threads)
```

Comment:
Can `Threads::Threads` be an exported dependency from `asio`?

Reply:
Sorry, didn't see this comment until now. To be honest, ASIO ought to never yank in pthreads anyway in C++ 11, it never used to in fact, only in this v1.16.1 was the dependency failing. v1.17.0 is out now, so might as well do the upgrade to that. I'll send a PR for v1.17.0 towards cpp-pm/asio shortly.

Reply:
https://github.com/cpp-pm/asio/pull/5

Reply:
@ned14 https://github.com/cpp-pm/asio/releases/tag/v1.17.0-p0